rosa_architecture/index.adoc

```diff
@@ -59,7 +59,7 @@ Start with xref:../architecture/architecture.adoc#architecture-overview-architec
 xref:../security/container_security/security-understanding.adoc#understanding-security[Security and compliance].
 ifdef::openshift-enterprise,openshift-webscale[]
 Next, view the
-xref:../release_notes/ocp-4-15-release-notes.adoc#ocp-4-15-release-notes[release notes].
+xref:../release_notes/ocp-4-18-release-notes.adoc#ocp-4-18-release-notes[release notes].
```

Comment:
While it won't hurt anything to make this update, just a heads-up that this file is not published anywhere.